Vector/EmbeddingGenerator/README.md

```diff
@@ -0,0 +1,175 @@
+---
+page_type: sample
+languages:
+- python
+products:
+- azure
+- azure-search
+- azure-cognitive-search
+- azure-openai
+name: Text chunker and embedding skill for Azure Cognitive Search
+description: The custom skill chunks content using the open source LangChain text chunker and then utilizes Azure Open AI service to generate vector embeddings for that content.
```

Comment:
Same as in top-level readme: add "the" and a link.

Reply:
Do we also want to add some links to basic definitions and/or tutorials about vectors? I'd also add a word about some end-to-end scenarios this skill will be useful for.

Reply:
The description is just metadata right? I'll add a link to the embedding model page...

Adding links/tutorials - can we do that as a different PR? We will be publishing a lot of vector content, and once that's out there, I can add those links

Reply:
(same for the end-to-end scenarios)